customer_satisfaction/README.md

```diff
@@ -29,9 +27,9 @@ zenml integration install mlflow -f
 
 ## :thumbsup: The Task
 
-Our team at ZenML was looking for a project which is model centric, data centric and zenml centirc for our next Zenfile. During our intiial discussions we listed out several questions like What will be the usecase for this zenfile? what's the impact of this zenfile?, etc. to evaluate a zenfile. We found out that it would be really great if we build an end to end pipeline for predicting the customer satisfaction score for the next order or purchase which will help businesses to take better decisions. 
+Our team at ZenML was looking for a project which is model centric, data centric and zenml centirc for our next Zenfile. During our intiial discussions we listed out several questions like What will be the usecase for this zenfile? what's the impact of this zenfile?, etc. to evaluate a zenfile. We found out that it would be really great if we build an end to end pipeline for predicting the customer satisfaction score for the next order or purchase which will help businesses to take better decisions. We can't just train our model in our local system and we need to serve to the users, so we need it to be deployed in the cloud. For doing Machine learning at scale we need machine learning pipelines which is an end-to-end construct that orchestrates the flow of data into, and output from, a machine learning model (or set of multiple models). It includes raw data input, features, outputs, the machine learning model and model parameters, and prediction outputs. and All these capbilities are built on top of the zenml framework.
```

Comment:
I think following on from @AlexejPenner's point, I think you can maybe just focus on the problem itself rather than talking about ZenFiles or ZenML or our internal discussion. I'm not sure that readers care about our internal discussion, but they do care about the actual problem you're trying to solve.